index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html ng-app="MyTodoApp">
+<head>
+  <title>First TO-DO MVP</title>
+
+  <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css" />
+  <link rel="stylesheet" href="public/stylesheets/main.css" />
+</head>
+
+<body>
+  <div class="container" ng-controller="TodoListController as todoCtrl">
+    <div class="row">
+      <div class="col-md-6">
+        <div class="todolist not-done">
+          <h1>To-Do (or Not-To-Do) List</h1>
+
+          <input type="text" class="form-control add-todo" placeholder="Add todo" ng-model="todoCtrl.newTask">
+          <button class="btn" ng-click="todoCtrl.addTask(todoCtrl.newTask)">Add Task</button>
+          <ul id="sortable" class="list-unstyled">
+            <li class="ui-state-default" ng-repeat="task in todoCtrl.todoList" ng-click="todoCtrl.completeTask(task, $index)">
+              {{ task }}
+            </li>
+          </ul>
+
+          <div class="todo-footer">
+            There is a total of {{ todoCtrl.todoList.length }} tasks to do.
+          </div>
+        </div>
+      </div>
+
+      <div class="col-md-6">
+        <div class="todolist">
+          <h1>Proudly Done</h1>
+          <ul class="list-unstyled done-items">
+            <li ng-repeat="task in todoCtrl.completedList">{{ task }}
+              <button class="remove-item btn btn-default btn-xs pull-right">
+                <span class="glyphicon glyphicon-remove"></span>
+              </button>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <script type="text/javascript" src="bower_components/angular/angular.js"></script>
+  <script type="text/javascript" src="bower_components/angular-resource/angular-resource.js"></script>
+  <script type="text/javascript" src="public/scripts/app.js"></script>
```

Comment:
Since you love Grunt soooooo much, why not minify all your JS into a single file?